src/public_html/desktop_settings.html

```diff
@@ -75,6 +75,10 @@ <h5 class="theme-text">Choose your theme highlight color</h5>
                 <input type="checkbox" class="filled-in" id="playback-api" />
                 <label for="playback-api">Enable Playback API <i>This allows other applications to interface with GPMDP</i></label>
               </p>
+              <p>
+                <input type="checkbox" class="filled-in" id="native-frame" />
+                <label for="native-frame">Use system window decorations <i>Requires restart.</i></label>
```

Comment:
Can we change this to `Use system window border styles`.  The average user might understand that a bit more?